src/components/organisms/Topnavigation/Topnavigation.tsx

```diff
@@ -52,8 +44,18 @@ const TopNavigation = (props: TopnavigationProps) => {
     };
   }, [Ref]);
 
+  {
+    isMenuOpen || isSubMenuOpen
```

Comment:
Place this block(line 47-51) inside useEffect and add referenced variables to dependency array, we don't want to fire these on each render

Reply:
Solved